masonry/src/widgets/grid.rs

```diff
@@ -113,8 +108,13 @@ fn new_grid_child(params: GridParams, widget: WidgetPod<dyn Widget>) -> Child {
 
 // --- MARK: IMPL GRIDPARAMS ---
 impl GridParams {
-    #[expect(missing_docs, reason = "TODO")]
+    /// Get grid parameters with the given values.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the width or height is less than or equal to zero.
```

Comment:
This also panics if `x` or `y` are negative, right? Is there some reason this shouldn't also be called out that I'm missing.